src/Evolution/DiscontinuousGalerkin/Limiters/HwenoImpl.hpp

```diff
@@ -556,5 +558,55 @@ void hweno_modified_neighbor_solution(
   }
 }
 
+// Implement the HWENO limiter for one tensor
+//
+// This interface is intended for use in limiters that check the troubled-cell
```

Comment:
Sorry, what part of the interface? The `PackagedData`? The way 561 and 563 read together to me is "This does A." and "This function is designed for B," which leaves me confused about which part of the comment is correct (presumably both) but I'm having trouble reconciling the two. Help? :P

Reply:
In the simple WENO case, I had two functions: one that applied the limiter to a single tensor component, and one that applied the limiter to all components of a tensor (by looping over the former). In that case, I was distinguishing between these by describing an interface for limiters that apply the TCI to each tensor component separately, vs an interface for limiters that apply the TCI to the whole cell.

Here I just used the same wording... but as there is only one function, perhaps the word "interface" is not ideal? Do you think it would be clearer if I wrote "This function is intended..." ?

Reply:
Yes, that would help, but I'm not sure it addresses the underlying confusion for me. If this function limits one tensor, then why not just stick to:
```cpp
// Implement the HWENO limiter for one tensor.
// Note, however, that HWENO will limit all variables if any of them
// are troubled. The function signature was chosen to be similar to
// that of SimpleWENO and to allow the flexibility of limiting only one
// tensor.
```
Is that accurate?

Reply:
Yeah, that's accurate. But it's also moot, because the commit that removes `hweno_modified_neighbor_solution` also vastly expands on this documentation (reusing the extensive dox of the now-removed function).

Can you look at the later commit and see if that is still ok to you?

I apologize for not identifying sooner that we were discussing something temporary 😞

Reply:
:+1: Sounds good to me :) I'm happy with this as-is then. Marking the convo as resolved :)